### DISCUSSION TITLE
School issued email address doesn’t get verified

### DISCUSSION THREAD

**Discussion by yoshihiro0323** [community/community#101926] [GitHub Education] [answered]
### Hi everyone,

My University issued email address doesn’t get verified.
This year, the domain has changed and those who entered this year, which includes me, have email addresses with different domain from verified ones
How can I fix this issue?
![IMG_5763](https://github.com/community/community/assets/87298805/4594c398-0257-493e-8ff5-671ca9897a9b)

**Comment by juanpflores** [comment, marked as answer]
Hey there! I can see you got a suppor ticket open. For this issue. I'll follow up in that ticket instead. If your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic email, then please provide supporting documentation such as:

- A link to a publicly-accessible page on the school website which lists their academic domain(s)
- An affidavit from your school on school letterhead detailing their current academic email issuing policy
